.travis.yml

```diff
@@ -1,6 +1,23 @@
+sudo: false
+
 language: java
+
+notifications:
+  email: false
+
+env:
+  global:
+    - PATH="/tmp/proto2.4.1/bin:$PATH"
```

Comment:
Okay. I've added PROTO_VERSION as an environment variable in the .travis.yml and use it instead of duplicating the version.